### PR TITLE
ENH: --out_itk option for LTAConvert

### PR DESCRIPTION
**Pull request by Gilles86**
Freesurfer's lta_convert can also convert to the ITK-format used by ANTS. This can be quite handy, for example when you have a small FOV and want to init a ANTS-registration based on a manual transform made with Freesurfer's Freeview.

This pull extends  https://github.com/nipy/nipype/commit/69a59a92250fccda81c0ac621e462c034cbdfe22 by @effigies: bby a few lines, so you can also use out_itk.

### CODE REVIEW

**Comment by effigies** [comment]
Actually, would you mind adding `--initk`, as well? Doc line:

```
	--initk <initk.txt>
		input ITK transform (inverse LPS2LPS). Convert ANTS .mat 
		binary to ITK format with ConvertTransformFile (in ANTS).
```

**Comment by Gilles86** [comment]
No. Just added it.

File: nipype/interfaces/freesurfer/tests/test_auto_LTAConvert.py [hunk 1]
```diff
@@ -14,23 +14,27 @@ def test_LTAConvert_inputs():
...
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
     ),
     in_reg=dict(argstr='--inreg %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
+    ),
+    in_reg=dict(argstr='--initk %s',
```
Comment (effigies) [inline nipype/interfaces/freesurfer/tests/test_auto_LTAConvert.py]: This should be `in_itk`. This file really shouldn't be edited by hand; run `make specs` and add the changes. (`make specs` does modify other files, so only commit the ones with changes.)

Comment (Gilles86) [reply nipype/interfaces/freesurfer/tests/test_auto_LTAConvert.py]: Whoops. Sorry, should have seen the comment on the first line. This kind of stuff is a bit new to me. :)

**Pull request merged**